src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -0,0 +1,260 @@
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(VCFConstants.ALLELE_COUNT_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(VCFConstants.ALLELE_FREQUENCY_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(VCFConstants.ALLELE_NUMBER_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(GATKVCFConstants.NOCALL_CHROM_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(VCFConstants.RMS_MAPPING_QUALITY_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(VCFConstants.MAPPING_QUALITY_ZERO_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(GATKVCFConstants.SAMPLE_LIST_KEY));
+        Assert.assertNull(resultVC.getCommonInfo().getAttribute(GATKVCFConstants.STRAND_ODDS_RATIO_KEY));
+    }
+
+    @Test
+    public void testCoverageAnnotationOnDbSnpSite() throws Exception {
+        final List<String> annotationGroupsToUse= Collections.emptyList();
+        final List<String> annotationsToUse = Arrays.asList(Coverage.class.getSimpleName());//good one
+        final List<String> annotationsToExclude= Collections.emptyList();
+        final File file= new File(publicTestDir + "Homo_sapiens_assembly19.dbsnp135.chr1_1M.exome_intervals.vcf");
+        final FeatureInput<VariantContext> dbSNPBinding = new FeatureInput<>("dbsnp", Collections.emptyMap(), file);
+
+        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofSelectedMinusExcluded(annotationGroupsToUse, annotationsToUse, annotationsToExclude, dbSNPBinding);
+
+        final Set<VCFHeaderLine> vcfAnnotationDescriptions = vae.getVCFAnnotationDescriptions();
+        Assert.assertTrue(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.DBSNP_KEY)));
+
+        final int alt = 5;
+        final int ref = 3;
+
+        final SimpleInterval loc= new SimpleInterval("1", 69428, 69428);
+        final VariantContext vcRS = new FeatureDataSource<>(file, new VCFCodec(), null, 0).query(loc).next();
+
+        final Allele refAllele = vcRS.getReference();
+        final Allele altAllele = vcRS.getAlternateAllele(0);
+
+        final VariantContext vcToAnnotate = makeVC(refAllele, altAllele, loc);
+        final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap = makeReadMap(ref, alt, refAllele, altAllele, loc.getContig(), loc.getStart() - 5);
+
+        final FeatureContext featureContext= when(mock(FeatureContext.class).getValues(dbSNPBinding, loc.getStart())).thenReturn(Arrays.<VariantContext>asList(vcRS)).getMock();
+        final VariantContext resultVC = vae.annotateContext(featureContext, null, vcToAnnotate, perReadAlleleLikelihoodMap, a->true);
+        Assert.assertEquals(resultVC.getCommonInfo().getAttribute(VCFConstants.DEPTH_KEY), String.valueOf(ref+alt));
+        Assert.assertEquals(resultVC.getID(), vcRS.getID());
+    }
```

Comment:
I don't think this is even enabled now. I'll enter this: https://github.com/broadinstitute/gatk/issues/1485